### ISSUE TITLE
What is the final licence of the project?

### ISSUE THREAD

**Issue by brcinho** [thomasgalliker/ValueConverters.NET#25]
Github depicts MIT, Nuget repo says Apache 2.0, which both allow free for commercial use with attribution and mention of the author.
On the other hand on the bottom of Readme/Home page on Github, it is written that for commercial use, contact the author?

Can you provide some clarification please?

**Comment by martin-braun** [comment]
Yes, please give some clarification. I would suggest MIT. I happily recommend this package to any WPF developers, but I don't think my customers care about what packages I use in my programs.

**Comment by thomasgalliker** [comment]
I'll be looking into this issue since it raises a lot of questions. I want to avoid a situation where I maintain dozens of actively used nugets as my fulltime job without getting a single coin back. The truth is, big companies use this dual licensed nugets in their products without giving shit about the licensing. I know it, because I saw it in customer projects. Pretty sad.

**Comment by martin-braun** [comment]
@thomasgalliker I really appreciate the work you put into this project and I agree that open source contributions should be honored more in general, that's why I decided to add you to my sponsor list, although it's only a little start. 

I would really wish for a commercially free license for this project, because your package is so mandatory for WPF. I don't get why the converters here are not part of the feature-set of WPF in the first place.

Thank you.
